shared/p2p/discovery.go

```diff
@@ -32,6 +35,24 @@ func startDiscovery(ctx context.Context, host host.Host) error {
 	return nil
 }
 
+// startDHTDiscovery supports discovery via DHT.
+func startDHTDiscovery(ctx context.Context, host host.Host, bootstrapAddr string) error {
+	addr, err := iaddr.ParseString(bootstrapAddr)
+	if err != nil {
+		return err
+	}
+	peerinfo, err := ps.InfoFromP2pAddr(addr.Multiaddr())
+	if err != nil {
+		return err
+	}
+
+	// Wrap the host with a routed host.
+	dht := kaddht.NewDHT(ctx, host, dsync.MutexWrap(ds.NewMapDatastore()))
+	host = rhost.Wrap(host, dht)
```

Comment:
Oh wait, you already do this... Sorry 😳  please remove

shared/p2p/service.go L52

Reply:
Hi @prestonvanloon which line should I remove? 
No more routed host, just the following?
```
dht := kaddht.NewDHT(ctx, host, dsync.MutexWrap(ds.NewMapDatastore())) // REMOVE
host = rhost.Wrap(host, dht) // REMOVE
err = host.Connect(ctx, *peerinfo)
```

Reply:
Correct

Reply:
Correct, just the routed host part